ui/src/app/components/runtime-configuration-panel/disabled-panel.tsx

```diff
@@ -7,7 +7,7 @@ import { AoU } from 'app/components/text-wrappers';
 export const DisabledPanel = () => {
   return (
     <WarningMessage
-      data-test-id='runtime-disabled-panel'
+      data-test-id='environment-disabled-panel'
```

Comment:
please update the text below here: notebooks -> notebooks or user apps, or similar